src/test/groovy/ratpack/zipkin/internal/ZipkinHttpClientImplSpec.groovy

```diff
@@ -39,34 +46,136 @@ class ZipkinHttpClientImplSpec extends Specification {
 				.build()
 
 		zipkinHttpClient = new ZipkinHttpClientImpl(httpClient, tracer, new DefaultSpanNameProvider())
+
+		httpClient.request(_ as URI, _ as Action) >> { URI u, Action<? super RequestSpec> a ->
+			a.execute(requestSpec)
+			return Promise.value(this.receivedResponse)
+		}
+		requestSpec.get() >> requestSpec
 	}
+	@Unroll
+	def "#method requests should be traced"(HttpMethod method) {
+		given:
+			receivedResponse.getStatusCode() >> 200
+			requestSpec.getUri() >> uri
+		when:
+			harness.yield { e ->
+				zipkinHttpClient.request(uri, {spec -> spec.method(method)})
+			}
+		then:
+			reporter.spans.size() == 1
+			Span span = reporter.spans.get(0)
+			span.name == method.name.toLowerCase()
+		and: //should contain the correct annotations
+			assertThat(span.annotations.findAll {it.value == "cs"}).isNotEmpty()
```

Comment:
nitpick; there are constants for these https://github.com/openzipkin/zipkin/blob/master/zipkin/src/main/java/zipkin/Constants.java#L34